runtime/codert_vm/cnathelp.cpp

```diff
@@ -213,6 +213,24 @@ restoreBranchJITResolveFrame(J9VMThread *currentThread)
 	restoreJITResolveFrame(currentThread, NULL, false, false);
 }
 
+VMINLINE bool
```

Comment:
you dont need to copy this function, I didnt realize this is what you were talking about. Yes, your original suggestion of moving this into VMHelpers.hpp is correct. This way it can be shared by the interpreter and the runtime helpres